client-api/src/main/java/org/sonarsource/sonarlint/core/clientapi/backend/tracking/ClientTrackedFindingDto.java

```diff
@@ -23,7 +23,7 @@
 import javax.annotation.CheckForNull;
 import javax.annotation.Nullable;
 
-public class ClientTrackedIssueDto {
```

Comment:
FYI, with the new CCT, the only concept left will be "issues". "findings" was considered but not retained. We can keep "finding" as an internal concept for now, as long as we still have hotspots.

Reply:
That's good to know, thanks for sharing!